server/kafka/kafkaproducer.go

```diff
@@ -0,0 +1,69 @@
+package kafka
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/cosminrentea/gobbler/server/configstring"
+	"github.com/cosminrentea/gobbler/server/service"
+	"time"
+)
+
+type Producer interface {
+	service.Startable
+	service.Stopable
+	Report(topic string, bytes []byte, key string)
```

Comment:
As a suggestion here. Wouldn't it be nice if we use a io.ReadCloser as the body? If possible :dancer: